modules/library/src/main/scala/zio/elasticsearch/ElasticPrimitive.scala

```diff
@@ -39,6 +39,10 @@ object ElasticPrimitive {
     def toJson(value: Double): Json = Num(value)
   }
 
+  implicit object ElasticFloat extends ElasticPrimitive[Float] {
```

Comment:
Just keep in mind when you start with rebase later that you have already added this in other PR.